### ISSUE TITLE
Evaluation/testing pipeline uses outdated configuration dimensions

### ISSUE THREAD

**Issue by MagicWiqqu** [AI-MED-AGH/robust-radiotherapy-planning#37]
The evaluation/testing pipeline configuration is inconsistent with the current expected setup. Requested changes:

* Update the evaluation/testing pipeline configuration to use 128×128×32 instead of 64×64×16.
* Verify that all evaluation and testing jobs reference the updated configuration.
* Set the latent scale to be consistent with the one set in training.
* Add the option to run the pipeline on the validation set instead of the test set.
* Make sure the evaluation pipeline uses the class label `2` for the U-Net instead of `0`.
